### ISSUE TITLE
make api.vilfredo.org work

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#138]
As in #74 we need to use the subdomain https://api.vilfredo.org to make API calls (and be able to measure them later, and be indipendent from the client).
As of now, the API (core) is in a subfolder of Vilfredo.

**Comment by pietrosperoni** [comment]
@giovanisp Can @derekpa just copy the info from the folder /api into api.vilfredo.org?
Because on #74 I was asked to make a mirror, but on Dreamhost mirrors only permit to mirro a subdomain to another. Not to a folder to a subdomain

**Comment by giovanisp** [comment]
Yes he can, although I think this will break some things for him. Also, the authentication will be no longer secure, until we have a https certificate on api.vilfredo.org as well. So I suggest moving this to 2.0.1.

**Comment by pietrosperoni** [comment]
ok, moved this to 2.0.1
